src/components/pages/AccountSettingsForm/ChangePasswordModal.js

```diff
@@ -3,7 +3,7 @@ import { Modal } from 'antd';
 import { LockOutlined } from '@ant-design/icons';
 
 const ChangePasswordModal = props => {
-  const [visible, setVisible] = useState(false);
+  const [, setVisible] = useState(false);
```

Comment:
Is the state "visible" missing here?

Reply:
No reason to have empty declarations in the code. It throws warnings and causes console clutter. Also, it's just bad form to leave unused imports and declarations, especially without accompanying comments to explain why they are there if for some reason they need to be - sort of like leaving trash behind.